game-core/src/main/java/games/strategy/engine/random/MartiDiceRoller.java

```diff
@@ -35,12 +35,15 @@
 import org.apache.http.util.EntityUtils;
 
 /** A pbem dice roller that reads its configuration from a properties file. */
-@Log
 @Builder
 public final class MartiDiceRoller implements IRemoteDiceServer {
   private static final int MESSAGE_MAX_LENGTH = 200;
   private static final String DICE_ROLLER_PATH = "/MARTI.php";
 
+  private final Pattern errorPattern = Pattern.compile("fatal error:(.*)!");
+  private final Pattern dicePattern =
+      Pattern.compile("your dice are:\\s*((?:\\d(?:,\\d+)*)?)\\s*<p>");
```

Comment:
Any objection to adding a comment to explain this regex a bit and give an example of what it matches?
Can we simplify the regex, does it have to be this long or maybe strict? It is a bit scary looking and seems to be more specified than what we had. I'm not sure if we need it to be extremely "strict".

Reply:
The "benefit" of this regex being rather strict is that it allows us to avoid any further exception handling down below.
So we could of course simply it quite a bit, but would then have to re-introduce a try catch block below that rethrows an exception.

But yeah adding a comment is a good idea